aat/befta_recent_executions_info.json

```diff
@@ -0,0 +1 @@
+{"lastExecutionTime":"2020-11-06T11:28:23.212","lastExecutionProjectRepo":"origin\tgit@github.com:hmcts/ccd-case-print-service.git (fetch)","lastExecutionProjectBranch":"RDM-8934-timeBasedReexecutionOfTestData"}
```

Comment:
This file shouldn't be in any PRs. This is not part of repo.
Also, the lastExecutionProjectRepo doesn't look correct. Just `ccd-case-print-service` alone could look a lot nicer.

Reply:
One more thing is, we should generate this file in a nicely indented json format for convenience of its readers when they need.

Reply:
Also, we should merge this change with CI/CD. So, please branch of from master.